src/components/site-header/SiteHeaderMenu.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const SiteHeaderMenu = ({ enterpriseConfig }) => {
```

Comment:
nit: `SiteHeaderNavMenu` to differentiate it from the user profile menu in the header 🤔